package/cpp/api/JsiSkPath.h

```diff
@@ -42,15 +42,15 @@ class JsiSkPath : public JsiSkWrappingSharedPtrHostObject<SkPath> {
   }
 
   JSI_HOST_FUNCTION(addArc) {
-    auto rect = JsiSkRect::fromValue(runtime, arguments[0]).get();
+    auto rect = JsiSkRect::fromValue(runtime, arguments[0]);
```

Comment:
Not sure if I understand correctly what this fix does (or why it was failing) - but it seems ok to remove the `get()` ptr accessor here.